app/Resources/views/committee/timeline/_message.html.twig

```diff
@@ -1,5 +1,5 @@
 {% if is_granted('ROLE_ADHERENT') %}
-    <div class="text--body">{{ item.content|raw }}</div>
+    <div class="committee-event-description text--body">{{ item.content|raw }}</div>
```

Comment:
I agree that the current naming is confusing.

The styling is used in three different places:

- In the timeline for messages (app/Resources/views/committee/timeline/_message.html.twig)
- In the timeline for event descriptions (app/Resources/views/committee/add_event.html.twig)
- On the page of each event for the event description (app/Resources/views/events/show.html.twig)

The simple solution is to rename `committee-event-description` to something more general like `committee-feed-item-description` or 'feed-item-description`.

Another solution, which you suggested, would be to avoid using a class in the CSS selector, but then I'd need to move the styling in a mixin because it would be used in 2 or 3 different places.

What do you want me to do? ;-)